tensorflow_tts/processor/base_processor.py

```diff
@@ -178,7 +178,13 @@ def _load_mapper(self, loaded_path: str = None):
         self.symbol_to_id = data["symbol_to_id"]
         self.id_to_symbol = {int(k): v for k, v in data["id_to_symbol"].items()}
 
-    def _save_mapper(self, saved_path: str = None):
+        # other keys
```

Comment:
@machineko let take a look if it make sense :)).

Reply:
LGFM